ClientSecurity/Configuring_IPS_Notifications.md

```diff
@@ -0,0 +1,61 @@
+# Configuring Intrusion Prevention System (IPS) Notifications
```

Comment:
Remove the title. No need for the title of the posts to be included in the body because it's pulled from the front matter.